base/logging.c

```diff
@@ -663,7 +663,28 @@ gvm_log_func (const char *log_domain, GLogLevelFlags log_level,
           break;
         }
 
-      syslog (syslog_level, "%s", message);
+      /* Syslog doesn't support messages longer as 1kb. The overflow data
```

Comment:
This sentence "The overflow data the will not be logged" looks some kind of broken.